packages/core/src/style.css

```diff
@@ -1,7 +1,2 @@
-/*
-This is an empty placeholder file, which should NOT contain CSS code.
-It's here so DEV environment doesn't show a 404
-
-- In DEV environment, examples/editor loads this stub file, but actual CSS is loaded from CSS modules directly
-- In PROD environment, the actual dist/style.css file is built from the CSS modules
-*/
+@import url("./editor/Block.css");
```

Comment:
What's the reason for this change? Doesn't the CSS get generated when building anyway?

Reply:
I think there was a good reason but didn't document it. Reverting and will see when we run into this

Reply:
Ok, added it again. The issue with the old setup was that the playground example for static rendering imported the placeholder file in dev mode. The only way to fix that would be to include BlockNoteView, which then imports the right CSS, but we specifically don't want that in this example.

I don't think there are downsides to the fix